NEWS.md

```diff
@@ -44,6 +44,25 @@
 * `boolean_arithmetic_linter()` for identifying places where logical aggregations are more appropriate, e.g.
   `length(which(x == y)) == 0` is the same as `!any(x == y)` or even `all(x != y)` (@MichaelChirico)
 
+## Notes
+
+* `lint()` continues to support Rmarkdown documents. For users of custom .Rmd engines, e.g.
```

Comment:
@IndrajeetPatil PTAL. I moved it from "bug fixes" because there's no longer any code change by lintr. I still don't feel like I have a 100% grasp of the situation, so feel free to refine the item further.

Reply:
Thanks, Michael. This looks good to me.

I think we can revise this further as our understanding of the issue evolves while working on #1617.